site/docs/v1/tech/guides/migration.adoc

```diff
@@ -0,0 +1,583 @@
+---
+layout: doc
+title: Migration Guide
+description: How to migrate your users to FusionAuth.
+---
+:page-liquid:
+
+== Overview
+
+This guide will help you migrate existing users to FusionAuth. It covers the types of migrations available as well as the phases and activities of a successful migration.
+
+== Types of Migrations
+
+There are three approaches to user data migration. 
+
+* Migrate everyone at once, also known as a "big bang" migration.
+* Segment your users and migrate each segment in a series of small bang migrations. 
+* Migrate user data each time a user authenticates, also known as "slow migration".
```

Comment:
Are the first two types really the same, just batching them into smaller batches? 

The types we've seen:

1. All at once (big bang)
   - This may be all in a single import, or 1-to-many imports but the cut-over is still all at once. 
2. Initial import, and then additional imports over the next weeks or months
  - This may occur for a slower migration across apps, or business units, but it still uses the strategy of "all at once" for the users being migrated. 
3. Slow migration - (as you have stated)

Reply:
I think that your point 2 and my segment by segment are the same things. When is cutover for the initial import? Additional imports? That's the key difference is that with a segment by segment implementation you have multiple cutovers (to be planned for). 

If I didn't capture what you meant by point 2, then maybe we should chat.

Reply:
Ok, got it. Perhaps the confusion (for me) is the time element. You are implying in your second bullet that the segments are separated by time. 

Maybe just clarify that, and I think we are on the same page.